kotlin/src/androidTest/java/app/rive/runtime/kotlin/core/RiveViewTest.kt

```diff
@@ -452,7 +452,7 @@ class RiveViewTest {
             view.setRiveResource(R.raw.multiple_animations, autoplay = false)
 
             view.play("one", Loop.PINGPONG)
-            view.drawable.advance(100f)
+            view.drawable.advance(0.1f)
```

Comment:
Yeah, there's a few of these that are still failing that I need to look into. 
I made this one change because we're advancing in `ms` now instead of `ns`, which we previously converted dividing by `1000`